portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+
+/** Class used to create object to store comment data */
```

Comment:
Class comments are similar to method comments where you want to start with a verb. Also, we know its a class from line 21, so that can be omitted.

Here is more information on how to write class comments.
https://developers.google.com/style/api-reference-comments#classes_interfaces_structs

Reply:
Done